ui/lib/kmip/addon/templates/components/edit-form-kmip-role.hbs

```diff
@@ -0,0 +1,80 @@
+{{#if model.canDelete}}
+  <Toolbar>
+    <ToolbarActions>
+      <ConfirmAction
+        @buttonClasses="toolbar-link"
+        @onConfirmAction={{action (perform save model (hash method="destroyRecord"))}}
+        @confirmMessage="This config will be permanently deleted."
```

Comment:
is this supposed to be a config, or should it be 'role'?

Reply:
Oops, these are from copy and pasting the edit-form, but the if block is never true here - this will be what gets hooked up when I do delete stuff. I'll remove it from here though - thanks for catching!

Reply:
✂️ in the latest!